.repo-metadata.json

```diff
@@ -4,11 +4,12 @@
     "product_documentation": "",
     "client_documentation": "https://github.com/googleapis/python-test-utils",
     "issue_tracker": "https://github.com/googleapis/python-test-utils/issues",
-    "release_level": "beta",
+    "release_level": "preview",
     "language": "python",
     "library_type": "OTHER",
     "repo": "googleapis/python-test-utils",
     "distribution_name": "google-cloud-testutils",
     "default_version": "",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "google-cloud-test-utils"
```

Comment:
👋  The doc states `api_shortname` is only required for GAPIC* libraries. Is there different guidance somewhere else?

Reply:
Fixed.